css/style.css

```diff
@@ -0,0 +1,551 @@
+}
+.product-image img{
+    height: 80px;
+    width: 80px;
+    border-radius: 50%;
+    border: 4px solid #ffffff;
+    margin-top: -20px;
+}
+.card-body{
+	padding: 10px 0;
+}
+
+.card-body li p.runs{
+    font-size: 26px;
+    color: rgb(54, 54, 54);
+    position: relative;
+}
+.card-body li p.runs::after{
+    content:' ';
+    height: 2%;
+    width: 70%;
+    position: absolute;
+    top: 40px;
+    right: 45px;
+    background-color: rgb(226, 225, 225);
+
+}
+.card-body li p.product{
+    font-size: 26px;
+    color: rgb(54, 54, 54);
+    position: relative;
+}
+.card-body li p.product{
+    font-size: 26px;
+    color: rgb(54, 54, 54);
+    position: relative;
+}
+.deactivate-button a{
+    padding: 2px 16px;
+    background-color: #d6d5d5;
+    color: #ffffff;
+    border-radius: 3px;
+    margin-right: 12px;
+}
+.activate-button a{
+    padding: 2px 16px;
+    background-color:#90caf9;
+    color: #3b3b3b;
+    border-radius: 3px;  
+    position: relative;
+}
+.temperature{
+    width: 50%;
+    float: left;    
+    border-right: 1px solid #b3b3b3;
+    line-height: 1.2rem;    
+}
+
+.temperature span{
+    font-size: 20px;
+    color: #333333;
+}
+.card-body li{
+	margin: 10px 0;
+	font-weight: 400;
+    font-size: 15px;
+    line-height: 1.6;
+    color: #606060;
+}
+
+.card-footer{
+    /* padding: 40px 0px; */
+    border-top: 1px solid #747373;
+    margin-top: 60px;
+    padding: 15px 0;
+}
+
+.card-footer span{
+    padding: 7px 25px;
+    margin-right: 13px;
+    border-right: 1px solid#747373;
+}
+.card-footer span:nth-child(2){
+    border-right: none
+}
+
+  /* cards section ends */
+.cards{
+    float: left;
+    width: 96%;
+    text-align: center;
+    margin-left: 20px;
+    clear: both;
+    padding: 12px 0;
+}
+
+  
+@media screen and (max-width: 425px) {
+    .logo img{
+      width: 90px;
+      height: 30px;
+      margin-top: 6px;
+    }
+    .user a{
+        font-size: 13px;
+    }
+  }
+
+  @media screen and (max-width: 1024px){
+    .sidebar{
+        width: 30%;
+    }
+    .about-plan{
+        width: 78%;
+    }
+    .location img{
+        height: 52px;
+        width: 52px;
+    }
+    .detector img{
+        height: 51px;
+        width: 51px;
+    }
+    .plan-info p{
+        font-size: 17px;
+    }
+    .plan-right-content{
+        width: 70%;
+    }
+    .product-details li{
+        padding: 12px 28px;
+    }
+    .card{
+        margin: 0px 10px;
+    }
+    .card-footer span{
+        padding: 6px 11px;
+    }
+    .card-footer{
+        margin-top: 71px;
+    }
+  }
+
+  @media screen and (max-width: 768px){
+    .product-details li{
+        padding: 12px 13px;
+    }
+    .product-info span{
+        font-size: 18px;
+    }
+    p.product-head{
+        font-size: 14px;
+    }
+    .plans-date{
+        padding: 6px 0px;
+    }
+    .card{
+        width: 46%;
+        margin-top: 10px;
+    }
+    .sidebar{
+        height: 1234px;
+    }
+  }
+
+  @media screen and (max-width: 425px){
```

Comment:
@lakshmikhabbas Responsive media query needs to align properly in terms of Order or try to add media query in each section of CSS so that it is easy and understandable.